apps/src/code-studio/components/libraries/LibraryPublisher.jsx

```diff
@@ -62,13 +69,14 @@ export default class LibraryPublisher extends React.Component {
     libraryName: libraryParser.suggestName(
       this.props.libraryDetails.libraryName
     ),
-    libraryDescription: '',
-    selectedFunctions: {}
+    libraryDescription: this.props.libraryDetails.libraryDescription,
+    selectedFunctions: this.props.libraryDetails.selectedFunctions
   };
 
   setLibraryName = event => {
+    const {libraryName} = this.state;
     let sanitizedName = libraryParser.sanitizeName(event.target.value);
```

Comment:
here and elsewhere, prefer `const` over `let`

Reply:
✔️